### DISCUSSION TITLE
Inline documentation like Unraid and tooltips

### DISCUSSION THREAD

**Discussion by luzfcb** [gravitl/netmaker#2397] [Ideas]
Since we can always be inspired by the good ideas of other projects, I think Netmaker could be inspired by the inline documentation and tooltips used by [Unraid OS](https://unraid.net/) Web UI.

For example, let's assume I don't know what a certain setting is for. In Unraid OS, just click on the question mark icon and the inline documentation will be displayed.

https://github.com/gravitl/netmaker/assets/807599/5bfe1fd3-bcee-4dd0-98d2-fc1b09515845

Unraid also displays useful tooltips in various parts:

![image](https://github.com/gravitl/netmaker/assets/807599/3b6522a3-6a2e-4cf8-8685-053874a16866)

Netmaker can make use of similar approaches and thus facilitate the process of learning how to use/configure Netmaker.

**Comment by Aceix** [comment]
I agree. Something similar is acutually in the roadmap. Hopefully, it will soon be easier to navigate Netmaker.
For now, you can find how to navigate the dashboard here https://www.netmaker.io/demo-page
